index.html

```diff
@@ -227,35 +254,34 @@ <h1 class="about-title">About me</h1>
     <!-- Contact Form Start -->
 
 
-    <footer>
+    <footer id="footer">
 
         <div id="maincontact">
             <div class="contact">
-                <h2>I'm always <br> interested in hearing about new projects, so if you'd like to chat please get in
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in
                     touch.</h2>
             </div>
             <form action="https://formspree.io/f/mrgjwpjk" method="post" class="form-contact">
                 <div id="form-contact">
                     <input type="text" class="name" maxlength="30" required name="name" placeholder="Full name">
-
                     <input type="email" class="email" name="email" required placeholder="Email Address">
                     <textarea class="txt-area" maxlength="500" name="txt-area" required id="txt-area" cols="30" rows="5"
                         placeholder="Enter text here"></textarea>
-                </div>
-                <button class="form" type="submit"> Get in touch</button>
-                <div>
-                    <div id="c-line">
-                        <div></div>
-                    </div>
-                    <div id="s-icons">
-                        <img src="/images/social-media-icon.png" alt="">
-                    </div>
-                    <div class="l-line">
-                        <img src="/images/light.png" alt="">
-                    </div>
+                    <button class="form" type="submit"> Get in touch</button>
                 </div>
             </form>
         </div>
+        <div class="footer">
+            <div id="c-line">
+
+            </div>
+            <div id="s-icons">
+                <img src="/images/social-media-icon.png" alt="">
+            </div>
+            <div class="l-line">
+                <img src="/images/light.png" alt="">
+            </div>
```

Comment:
- [ ] Good job so far. However, I wonder where is this image located and what role is it playing? If I could guess, I think you wanted to add some spacing between the horizontal line and the social media icons, It really looks good but I think we can achieve the same result in a more appropriate way. This is by adding the `margin` property to the social media icons container and/or to the `<hr>` horizontal line tag(make sure you use it as suggested above) by doing this, we no longer need this div therefore, you will remove it and remain with clean codes.😃